<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Wares</class>
 <widget class="QDialog" name="Dlg_Wares">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>242</y>
     <width>316</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_save">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>18</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>商品编号：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lw_id"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>商品名：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lw_name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>计量单位：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lw_stroe_unit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>商品数量：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lw_amount"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>计价单位：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lw_sell_unit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>单价：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lw_price"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
